example/vanilla-k8s-block-driver/vsphere.conf

```diff
@@ -1,5 +1,6 @@
 [Global]
 cluster-id = "unique-kubernetes-cluster-id"
+volumemigration-cr-cleanup-intervalinhours = "24"
```

Comment:
Why does this need to be configurable? Volume migration CR is an internal resource that only CSI driver knows about/uses.

Reply:
volumemigration-cr-cleanup-intervalinhours => volumemigration-cr-cleanup-intervalinmin
24 => 120

Reply:
Setting this to 60